open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/handlers/ViewHandler.java

```diff
@@ -147,7 +157,7 @@ private void deleteEntity(EntityProxy proxy) {
         try {
             omEntityDao.purgeEntity(proxy);
         } catch (RepositoryErrorException | UserNotAuthorizedException | InvalidParameterException | EntityNotKnownException | EntityNotDeletedException | FunctionNotSupportedException e) {
-            log.error(e.getMessage(), e);
+            log.debug(e.getMessage(), e);
```

Comment:
I think error level is more meaningful here as an exception was thrown

Reply:
I will use the DataPlatformErrorCode here as use log.error will introduce logging forging security issue here.